### PR TITLE
Github workflow to consolidate scala steward PRs [BW-615]

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

**Review by evanbernstein — APPROVED** [review]
you want a rubber stamp approval, you got a rubber stamp approval!

**Review by rsasch — APPROVED** [review]

**Comment by cjllanwarne** [comment]
Bypassing tests and codecoverage because this is not production code.

File: .github/workflows/combine_scalasteward_prs.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: combine-scalasteward-prs
+
+on:
+  push:
```
Comment (cjllanwarne) [inline .github/workflows/combine_scalasteward_prs.yml:4]: Remove before merging!

**Review by Qi77Qi — APPROVED** [review]
🎉 :shipit:

File: .github/workflows/combine_scalasteward_prs.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          UNSUCCESSFUL_PRS=()
+
+          git fetch
+          git checkout develop
+          git checkout -B $BRANCH
+
+          while IFS= read -r pr
+          do
+            echo "Bringing in: $pr"
+            git fetch origin pull/${pr}/head:pr_${pr}
```
Comment (Qi77Qi) [inline .github/workflows/combine_scalasteward_prs.yml:40]: nice 👍 I guess this is a bit less work then `gh pr checkout xxx`

**Review by rsasch — APPROVED** [review]

**Pull request merged**